### PR TITLE
adding autogenerated_domain_namelabel_scope field

### PR DESCRIPTION
**Pull request by angouloure**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
container

File: src/azure-cli/azure/cli/command_modules/container/_params.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def load_arguments(self, _):
         c.argument('ports', type=int, nargs='+', default=[80], help='A list of ports to open. Space-separated list of ports')
         c.argument('protocol', arg_type=get_enum_type(ContainerNetworkProtocol), help='The network protocol to use')
         c.argument('dns_name_label', help='The dns name label for container group with public IP')
+        c.argument('auto_generated_domain_name_label_scope', arg_type=get_enum_type(AutoGeneratedDomainNameLabelScope), help='The autogenerate name label policy chosen by user')
```
Comment (wangzelin007) [inline src/azure-cli/azure/cli/command_modules/container/_params.py:69]: Please use options_list to give a short name if argument length > 22.

**Comment by wangzelin007** [comment]
Please follow https://aka.ms/submitAzPR to modify PR title.

File: src/azure-cli/azure/cli/command_modules/container/tests/latest/test_container_commands.py [hunk 2]
```diff
@@ -243,6 +243,41 @@ def test_container_create_with_msi_scope(self, resource_group, resource_group_lo
                          self.exists('ipAddress.ip'),
                          self.check('containers[0].image', '{image}')])
 
+
+    # Test create container using auto generated domain name label scope.
+    @live_only()
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/container/tests/latest/test_container_commands.py:248]: Why is this test marked as `@live_only()`?

**Comment by zhoxing-ms** [comment]
@angouloure Could you solve these comments and CI issues? Please note that if you do not resolve them before Thursday, this PR will not be able to catch up with the release of this sprint, we can only postpone it to the next sprint **(08-02)**

**Comment by zhoxing-ms** [comment]
@angouloure Could you solve these comments and CI issues in time? Please note that if you do not resolve them before Thursday, this PR will not be able to catch up with the release of this sprint, we can only postpone it to the next sprint (09-06)

**Pull request closed without merging**